build.gradle

```diff
@@ -94,9 +94,11 @@ allprojects {
                 mavenBom "org.junit:junit-bom:${junitVersion}"
             }
             dependencies {
+                dependency "org.hibernate.orm:hibernate-core:${hibernateCoreVersion}"
                 dependency "org.yaml:snakeyaml:${snakeYamlVersionOverride}"
                 dependency "io.netty:netty-codec:${nettyCodecVersionOverride}"
                 dependency "org.liquibase:liquibase-core:${liquibaseVersion}"
+                dependency "com.rabbitmq:amqp-client:${amqpCLientVersion}"
```

Comment:
Is adding rabbitmq to all projects necessary?

Reply:
That's not what this does. `dependencyManagement` specifies versions to use, IF they are declared in the project/module. I also declared the version explicitly on each module, but for some reason `app/gzac` chooses to ignore the declared version of our modules. Hence this 'hack'.